deploy/openshift.template.yaml

```diff
@@ -15,7 +15,6 @@ objects:
     annotations:
       ignore-check.kube-linter.io/minimum-three-replicas: "git-partition-sync-producer design does not adequately account for multiple replicas"
       ignore-check.kube-linter.io/no-readiness-probe: "git-partition-sync-producer does not receive traffic"
-      ignore-check.kube-linter.io/no-liveness-probe: "TODO: https://issues.redhat.com/browse/APPSRE-5919"
```

Comment:
Reason for removing?

Reply:
Bad copy-paste from vault-manager's template. I'd like dvo to create the tickets so I can assign myself. I don't think I can satisfactory ignore liveness check until metrics are implemented and the dvo tickets will remind me of that.